github/pulls.go

```diff
@@ -55,6 +57,7 @@ type PullRequest struct {
 	MaintainerCanModify *bool      `json:"maintainer_can_modify,omitempty"`
 	AuthorAssociation   *string    `json:"author_association,omitempty"`
 	NodeID              *string    `json:"node_id,omitempty"`
+	RequestedReviewers  []*User    `json:"requested_reviewers,omitempty"` // These are the current requested reviewers of the pull request
```

Comment:
I don't think this comment adds value, and think it should be removed.

Reply:
removed